src/logger/HttpLogger_test.go

```diff
@@ -0,0 +1,75 @@
+package logger
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//Testing HTTP Logger
+
+func TestCreateInstance(t *testing.T) {
+
+	//Creating a single instance
+	httpLogger := NewHttpLogger()
+	helper := GetTestHelper()
+	assert.NotNil(t, httpLogger)
+	assert.Equal(t, helper.mockAgent, httpLogger.Agent())
```

Comment:
you shouldn't need the test helper here you can just use `httpLogger.agent`, you are essentially testing the getter here.